scripts/data_guide/data_guide.qmd

```diff
@@ -280,9 +280,9 @@ Variants that are unable to be annotated by Genome Nexus are removed.
 
 [Gene symbol harmonization]{.underline}
 
-All submitted HUGO gene symbols will be harmonized against GRCh37 by checking the coordinates against the annotation. The below diagram illustrates how conflicts between submitted and harmonized symbols will be resolved.
+All submitted HUGO gene symbols will be harmonized against GRCh37 by checking the coordinates against the annotation.
 
--   Every single row of your bed file will be matched against the gene database. If the submitted symbol matches a row in the gene database and there is an overlap from the submitted bed region, the submitted symbol will be returned.
+-   Every single row of the bed file will be matched against the gene database. If the submitted symbol matches a row in the gene database and there is an overlap from the submitted bed region, the submitted symbol will be returned.
```

Comment:
Do we need to say how we build the gene database?

Reply:
How do we build the gene database?

Reply:
The database was created here: http://ftp.ensembl.org/pub/release-75/gtf/homo_sapiens/Homo_sapiens.GRCh37.75.gtf.gz